services/ui-src/index.html

```diff
@@ -1,52 +1,37 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="description" content="Medicaid Data Collection Tool" />
+    <meta name="theme-color" content="#ffffff" />
+    <title>Managed Care Reporting</title>
+    <script src="/env-config.js"></script>
+    <link rel="icon" href="/favicon.ico" />
+    <link rel="icon" type="image/png" href="/favicon-32x32.png" sizes="32x32" />
+    <link rel="icon" type="image/png" href="/favicon-16x16.png" sizes="16x16" />
+    <!-- Tealium Analytics Tag Manager -->
+    <script>
+      const tealiumEnvMap = {
+        "mdctmcr.cms.gov": "prod",
+        "mdctmcrval.cms.gov": "qa",
+      };
+      const tealiumEnv = tealiumEnvMap[window.location.hostname] || "dev";
+      const tealiumDomainMap = {
+        prod: "tags.tiqcdn.com",
+        qa: "tags.tiqcdn.com",
+        dev: "tealium-tags.cms.gov",
+      };
+      const tealiumDomain = tealiumDomainMap[tealiumEnv];
+      const tealiumPath = `https://${tealiumDomain}/cms-mdctmcr/${tealiumEnv}`;
+      document.write(`<script src="${tealiumPath}/utag.js" async><\/script>`);
+      document.write(`<script src="${tealiumPath}/utag.sync.js"><\/script>`);
+    </script>
+  </head>
 
-<head>
-  <meta charset="utf-8" />
-  <meta name="viewport" content="width=device-width, initial-scale=1" />
-  <meta name="description" content="Medicaid Data Collection Tool" />
-  <meta name="theme-color" content="#ffffff" />
-  <title>Managed Care Reporting</title>
-  <script src="/env-config.js"></script>
-  <link rel="icon" href="/favicon.ico" />
-  <link rel="icon" type="image/png" href="/favicon-32x32.png" sizes="32x32" />
-  <link rel="icon" type="image/png" href="/favicon-16x16.png" sizes="16x16" />
-  <!-- Tealium Analytics Tag Manager -->
-  <script>
-    var tealiumEnvMap = {
-      "mdctmcr.cms.gov": "prod",
-      "mdctmcrval.cms.gov": "qa",
-    };
-    var tealiumEnv = tealiumEnvMap[window.location.hostname] || "dev";
-    var tealiumUrl = `https://tags.tiqcdn.com/utag/cmsgov/cms-general/${tealiumEnv}/utag.sync.js`;
-    document.write(`<script src="${tealiumUrl}" async><\/script>`);
-  </script>
-</head>
-
-<body>
-  <noscript>You need to enable JavaScript to run this app.</noscript>
-  <!-- Tealium Analytics Tag Manager -->
-  <script>
-    window.utag_cfg_ovrd = { noview: true };
-  </script>
```

Comment:
I think we might still need this? I don't know what it does.

Reply:
According to the docs ( https://docs.tealium.com/platforms/javascript/settings/#how-it-works ) it prevents an automatic tealium call on initial page load. So actually I think we want to move this up ahead of the first script, so that it's guaranteed to be set before the main tealium script runs.